ParabolInc/parabol
# feat(prompt-to-join): Add teammate UI

Fixes #7802

<img width="1089" alt="image" src="https://user-images.githubusercontent.com/466991/233634316-6f5613d1-8487-476c-9bba-30865f6331a6.png">


**How to test:**

- Re-run the migration packages/server/postgres/migrations/1680865284180_addDomainJoinRequestTable.ts from previous PR, as I changed it (if you haven't)
- Delete existing `REQUEST_TO_JOIN_ORG` notifications 
```
r.db('actionDevelopment')
    .table('Notification')
   .filter({type:'REQUEST_TO_JOIN_ORG' })
   .delete()   
```

- Follow testing guide from  #8003 and #8044 to create necessary notifications
- Click on `Review` button in the notification in header, see that dialog appeared
- Click on `Review` button in the snackbar, see that dialog appeared
- Check that teams are listed correctly (see #7802)
